device-types/Cisco/WS-C6513-E.yaml

```diff
@@ -0,0 +1,47 @@
+---
+manufacturer: Cisco
+model: Catalyst 6513
+slug: ws-c6513
+part_number: WS-C6513
+u_height: 19
+is_full_depth: true
+subdevice_role: parent
+console-ports:
+  - name: con0
+    type: rj-45
```

Comment:
This should be part of the SUP card, should be modular and slotted in with the rest of the modules